app/models/concerns/geoblacklight/solr_document/citation.rb

```diff
@@ -1,30 +0,0 @@
-# frozen_string_literal: true
-
-module Geoblacklight
-  module SolrDocument
-    module Citation
```

Comment:
removing this constant (and it's methods) is a breaking change.  Do we need to do that?

Reply:
No, I can add it back in. I am not sure why it is breaking change though.